adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/CertificateAssertion/ClientAssertionCertificate.cs

```diff
@@ -74,25 +74,25 @@ public ClientAssertionCertificate(string clientId, X509Certificate2 certificate)
             this.Certificate = certificate;
         }
 
-		/// <summary>
-		/// Constructor to create credential with client Id and certificate and claimSet.
-		/// </summary>
-		/// <param name="clientId">Identifier of the client requesting the token.</param>
-		/// <param name="certificate">The certificate used as credential.</param>
-		/// <param name="additionalClaims">The additional claims to be added to the payload of the client assertion.</param>
-		public ClientAssertionCertificate(string clientId, X509Certificate2 certificate, Dictionary<string, string> additionalClaims) : this(clientId, certificate)
-	    {
-		    this.AdditionalClaims = additionalClaims ?? throw new ArgumentNullException(nameof(additionalClaims));
-	    }
+        /// <summary>
```

Comment:
There seem to be an indentation issue? spaces/tabs?
this does not make it obvious the code which really changed.

Reply:
I didn't really make any changes. In fact I didn't even mean to push this up... I was just creating a new local branch and this changelist had some changes. Sorry for the disturbance.